client/src/legacy/LeftAndMain.js

```diff
@@ -1508,7 +1508,13 @@ $.entwine('ss', function($) {
   });
 });
 
+// TODO: move this to an external util
```

Comment:
Please don't include `todo` comments - please either move it to an external util now, or just remove the comment - frankly if it's not done now it won't likely ever be done.

Reply:
Either of those is fine with me - but whichever way we go, it has to be done up front. If you're willing to put in the time to implement a way to escape strings across the board, great! If not (and I wouldn't blame you), please just remove the comment. We don't track todo comments, we track issues :p